harvester/terraform-provider-harvester
# Bump longhorn-manager and harvester version

related to https://github.com/harvester/harvester/issues/5524
### Description
- Bump longhorn-manager to v1.6.0
- Bump longhorn-manager to v1.3.0
- Remove redundant replace rule

### Test Plan
Setup: Create a Harvester cluster
Build terraform-harvester-provider
1. execute `make`
2. follow https://developer.hashicorp.com/terraform/cli/config/config-file#development-overrides-for-provider-developers to use terraform-harvester-provider from local build.    
      ```
      provider_installation {
        dev_overrides {
          "harvester/harvester" = "/home/brandboat/Code/harvester/terraform-provider-harvester/bin"
        }
        direct {}
      }
      ```
3. Add `export TF_CLI_CONFIG_FILE=~/.terraformrc` to `.bashrc`
4. Test terraform create VM https://gist.github.com/brandboat/5a5279d9c13604d2465da3ce0b277743
5. Run `terraform apply`

Check vm create successfully.
